source/mixins/versionable.js

```diff
@@ -0,0 +1,19 @@
+Space.messaging.Versionable = {
+
+  schemaVersion: 1,
+
+  onConstruction(data) {
+    if (_.isObject(data)) this._migrateToLatestVersion(data);
```

Comment:
Which I just realised that's also a breaking change I think we should make… `migrateFromVersion` => `transformFromVersion`. This aligns with the Meteor [Mongo.Collection](https://docs.meteor.com/#/full/mongo_collection) options API, and would therefore be a familiar concept